src/kes/table.py

```diff
@@ -276,6 +277,8 @@ def _serialize_field(self, field: Any, pb_field: pb_Field):
                     locPoint = LocationPoint(name=point.name, latitude=point.latitude,
                                              longitude=point.longitude, address=point.address)
                     pb_field.locations.elements.append(locPoint)
+            case date() as dateValue:
+                pb_field.date.FromDatetime(datetime.combine(dateValue, datetime.min.time()))
```

Comment:
There is also datetime.date listed here https://docs.python.org/3/library/datetime.html#available-types, perhaps that is a better fit?

Reply:
I tried using `datetime.date` but got this error : `AttributeError: 'datetime.date' object has no attribute 'utctimetuple'`
Hence, I have to convert it the above way.

Reply:
How are you constructing the `datetime.data` object?

Reply:
I am constructing the `datetime.date` object like this. for ex. : `date.today()`

Reply:
The reason why I have to use DateTime while serializing and deserializing is because Protobuf doesn't yet support `Date` type of message.